<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="battery">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
background: black;
 	border: 2px solid #2dcc6b;
}

QProgressBar::chunk {
background-color:  #2dcc6b;
}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="increaseBattery">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Increase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decreaseBattery">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrease</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hazardIndicatorOFF">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hazardIndicatorON">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lightsOFF">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="batteryWarning">
       <property name="text">
        <string>batteryWarning</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dayLights">
       <property name="text">
        <string>dayLights</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nightLights">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>nightLights</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="motorWarning">
       <property name="text">
        <string>motorWarning</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hornSignal">
       <property name="text">
        <string>hornSignal</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>

<widget class="QWidget" name="horizontalLayoutWidget_1">
  <property name="geometry">
    <rect>
      <x>80</x>
      <y>530</y>
      <width>500</width>
      <height>80</height>
    </rect>
  </property>

  <layout class="QHBoxLayout" name="horizontalLayout_1">
    <item>
      <widget class="QLabel" name="parkingSignal">
        <property name="text">
          <string>parkingSignal</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignCenter</set>
        </property>
      </widget>
    </item>

    <item>
      <widget class="QLabel" name="drivingSignal">
        <property name="text">
          <string>drivingSignalSignal</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignCenter</set>
        </property>
      </widget>
    </item>

    <item>
      <widget class="QLabel" name="neutralSignal">
        <property name="text">
          <string>neutralSignal</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignCenter</set>
        </property>
      </widget>
    </item>

    <item>
      <widget class="QLabel" name="reversingSignal">
        <property name="text">
          <string>reversingSignal</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignCenter</set>
        </property>
      </widget>
    </item>
  </layout>
</widget>

<widget class="QWidget" name="horizontalLayoutWidget_2">
  <property name="geometry">
    <rect>
      <x>703</x>
      <y>520</y> 
      <width>50</width>
      <height>50</height>
    </rect>
  </property>

  <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
      <widget class="QLabel" name="cruiseControlSignal">
        <property name="text">
          <string>cruiseControlSignal</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignCenter</set>
        </property>
      </widget>
    </item>
  </layout>
</widget>

   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>781</width>
      <height>90</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="batteryWarningON">
           <property name="text">
            <string>ON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="batteryWarningOFF">
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_1">
         <item>
          <widget class="QPushButton" name="dayLightsON">
           <property name="text">
            <string>ON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dayLightsOFF">
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="nightLightsON">
           <property name="text">
            <string>ON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nightLightsOFF">
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="motorWarningON">
           <property name="text">
            <string>ON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="motorWarningOFF">
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="hornSignalON">
           <property name="text">
            <string>ON</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hornSignalOFF">
           <property name="text">
            <string>OFF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
    </layout>
    </item>
    </layout>
   </widget>

    <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
        <rect>
          <x>80</x>
          <y>600</y>
          <width>500</width>
          <height>80</height>
        </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
          <widget class="QPushButton" name="parkingSignalON">
            <property name="text">
              <string>ON</string>
            </property>
          </widget>
        </item>

        <item>
          <widget class="QPushButton" name="drivingSignalON">
            <property name="text">
              <string>ON</string>
            </property>
          </widget>
        </item>

        <item>
          <widget class="QPushButton" name="neutralSignalON">
            <property name="text">
              <string>ON</string>
            </property>
          </widget>
        </item>

        <item>
          <widget class="QPushButton" name="reversingSignalON">
            <property name="text">
              <string>ON</string>
            </property>
          </widget>
        </item>
      </layout>
    </widget>

    <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
        <rect>
          <x>665</x>
          <y>581</y>
          <width>130</width>
          <height>80</height>
        </rect>
      </property>

      <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
          <widget class="QPushButton" name="cruiseControlON">
            <property name="text">
              <string>ON</string>
            </property>
          </widget>
        </item>

        <item>
          <widget class="QPushButton" name="cruiseControlOFF">
            <property name="text">
              <string>OFF</string>
            </property>
          </widget>
        </item>
      </layout>
    </widget>

   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>121</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QPushButton" name="leftIndicatorON">
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="leftIndicatorOFF">
       <property name="text">
        <string>OFF</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>147</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QPushButton" name="rightIndicatorON">
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rightIndicatorOFF">
       <property name="text">
        <string>OFF</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="mph">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MPH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="leftIndicator">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>leftIndicator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="speed">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>189</width>
      <height>179</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>125</number>
    </property>
   </widget>
   <widget class="QLabel" name="rightIndicator">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>rightIndicator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>440</y>
      <width>101</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QPushButton" name="increaseMPH">
       <property name="text">
        <string>Increase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="decreaseMPH">
       <property name="text">
        <string>Decrease</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="distance">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>125</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="range">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>125</number>
    </property>
   </widget>
   <widget class="QLabel" name="rangeLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>R:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="distanceLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
